app/components/operator-form/component.js

```diff
@@ -1,6 +1,24 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
+	// isIncluded: Ember.computed('model.include_in_changeset', function() {
+	// 	if (this.get('operator.include_in_changeset') === true) {
+	// 		return true;
+	// 	} else {
+	// 		return false;
+	// 	}
+	// }),
+	isIncluded: true,
+	actions: {
+		clickYes() {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).